src/controls/extras/treeitem.coffee

```diff
@@ -34,16 +34,25 @@ class TreeItem #extends Base
   handle_relevant_or_required: ->
     tree_div = @tree.tree_root
     current_node = tree_div.find("[node_value = '" + @value + "']")
-
-    current_node.attr('relevant', @node_relevant())
+    current_node.attr('item-required', @node_required())
```

Comment:
right, this is not part of the ECHO forms spec, but has been added to support SDPS use cases.  Not sure if we should update the spec or keep it as an extension.  I think the tree control is pretty useful and not specific to the original use cases (and I have been careful not to make it too specific)